testing/adios2/engine/ssc/TestSscBase.cpp

```diff
@@ -297,7 +297,8 @@ TEST_F(SscEngineTest, TestSscBase)
 
 int main(int argc, char **argv)
 {
-    MPI_Init(&argc, &argv);
+    int provided;
+    MPI_Init_thread(&argc, &argv, MPI_THREAD_MULTIPLE, &provided);
```

Comment:
I am not sure if we should initialize threads in every SSC test. These tests have nothing to do with SST, and therefore they have nothing to do with the MPI DP of SST. The SSC engine is designed to ensure there is a baseline staging method that we can use everywhere that has MPI and MPMD. Some of these environments do not provide multi-thread MPI. We should aim to ensure these tests work without multi-thread MPI.

Reply:
@JasonRuonanWang good observation, I changed all the tests `MPI_Init` in bulk with the idea of later providing a general solution, perhaps on the form of an wrapper named `ADIOS2_MPI_Init`.

In this specific case if the system does not have THREAD_MULTIPLE` MPI it will not fail it will simply return the enabled mode in the provided argument and keep the execution.

Reply:
@vicentebolea Yeah, I understand that if MPI does not allow multi-thread it will still successfully initialize, only returning a provided=0 or something. But still, if 'unfortunately', all of our CI virtual machines support multi-thread MPI, we will lose the chance to discover any issues that break the non-multi-thread routine.

Reply:
>  if 'unfortunately', all of our CI virtual machines support multi-thread MPI, we will lose the chance to discover any issues that break the non-multi-thread routine.

That is a good point.